recipes/sfml/all/conanfile.py

```diff
@@ -69,7 +74,18 @@ def requirements(self):
             if Version(self.version) >= "2.6.0":
                 self.requires("minimp3/cci.20211201")
 
+    @property
+    def _min_cppstd(self):
+        if Version(self.version) >= "3.0.0":
+            return 17
+        elif Version(self.version) >= "2.6.0":
+            return 11
```

Comment:
Version 2.6 and prior requires C++03, not C++11.

Reply:
Have I misread?

The  code does what you say, with the exception of being inclusive of 2.6.0. That said - the prior version did the same for 2.6.0.